packages/TorneloScoresheet/node_modules/react-native-pincode-input-component/src/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-native-pincode-input-component';
```

Comment:
It's a bit strange to check `node_modules` into the repository - since this is basically a build artifact. Do we need this here for some reason?

Reply:
yeah, we had to make it, it wasn't there by default so we figured it was easier to just push it instead of everyone needing to do it individually